app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -365,6 +397,12 @@ public void closeSurveyFragment(){
         }
     }
 
+    public void closeReviewFragment(){
```

Comment:
@idelcano Again more comments, I see closeSurveyFragmentAndAskReview, closeSurveyFragment, closeReviewFragment and few other methods without any comments.  (I am aware some of these methods were already there before this PR)